Meta.Tests.ps1

```diff
@@ -432,54 +435,88 @@ Describe 'Common Tests - Validate Example Files' -Tag 'Examples' {
+            }
+
+            Copy-item -Path $env:APPVEYOR_BUILD_FOLDER -Destination $powershellModulePath -Recurse -Force
         }
 
-        $exampleFiles = Get-ChildItem -Path (Join-Path -Path $moduleRootFilePath -ChildPath 'Examples') -Filter "*.ps1" -Recurse
-        foreach ($exampleFile in $exampleFiles)
+        $exampleFile = Get-ChildItem -Path (Join-Path -Path $moduleRootFilePath -ChildPath 'Examples') -Filter "*.ps1" -Recurse
+        foreach ($exampleToValidate in $exampleFile)
         {
-            Context -Name $exampleFile.Name {
+            $exampleDescriptiveName = Join-Path -Path (Split-Path $exampleToValidate.Directory -Leaf) -ChildPath (Split-Path $exampleToValidate -Leaf)
 
-                try
-                {
-                    $exampleError = $false
-                    $path = $exampleFile.FullName
-                    . $path
-
-                    $command = Get-Command Example
-                    $params = @{}
-                    $command.Parameters.Keys | Where-Object { $_ -like "*Account" -or $_ -eq "Passphrase" } | ForEach-Object -Process {
-                        $params.Add($exampleFile, $mockCredential)
-                    }
-                    $null = Example @params -OutputPath "TestDrive:\" -ErrorAction Continue -WarningAction SilentlyContinue
-                }
-                catch
-                {
-                    Write-Warning -Message "Unable to compile MOF for example '$path'"
-                    Write-Warning $_.Exception.Message
-                    $exampleError = $true
-                }
+            Context -Name $exampleDescriptiveName {
+                It "Should compile MOFs for example correctly" -Skip:(!$optin) {
+                    {
+                        $mockPassword = ConvertTo-SecureString '&iPm%M5q3K$Hhq=wcEK' -AsPlainText -Force
+                        $mockCredential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList @('username', $mockPassword)
+                        $mockConfigurationData = @{
+                            AllNodes = @(
+                                @{
+                                    NodeName = 'localhost'
+                                    PSDscAllowPlainTextPassword = $true
+                                }
+                            )
+                        }
+
+                        try
+                        {
+                            . $exampleToValidate.FullName
+
+                            $exampleCommand = Get-Command -Name Example -ErrorAction SilentlyContinue
+                            if ($exampleCommand)
+                            {
+                                    $params = @{}
+                                    $exampleCommand.Parameters.Keys | Where-Object { $_ -like '*Account' -or ($_ -like '*Credential' -and $_ -ne 'PsDscRunAsCredential') -or $_ -like '*Passphrase' } | ForEach-Object -Process {
```

Comment:
Is there a reason to exclude PsDscRunAsCredential? I use that frequently in my examples I'm working on.